internal/gojs/compiler_test.go

```diff
@@ -96,6 +102,19 @@ func TestMain(m *testing.M) {
 		log.Panicln(err)
 	}
 
+	// In order to avoid race condition on scheduleTimeoutEvent, we need to set the memory max
+	// and WithMemoryCapacityFromMax(true) above.
+	// https://github.com/tetratelabs/wazero/issues/992
+	//
+	// TODO: Maybe add WithMemoryMax API?
+	parsed, err := binaryformat.DecodeModule(testBin, api.CoreFeaturesV2, wasm.MemoryLimitPages, false, false, false)
+	if err != nil {
+		log.Panicln(err)
+	}
+	parsed.MemorySection.Max = 1000
```

Comment:
This is a large amount of memory, and noting the issue number is nice, but maybe we should say what's going on.

```suggestion
	// Set max to a high value, e.g. so that Test_stdio_large can pass
	parsed.MemorySection.Max = 1024 // 64MB
```